### PR TITLE
Create css3d_youtube2.html

### PR DESCRIPTION
**Pull request by jsladek**
I stripped down the css3d_youtube.html example and thought it might be helpful to others.

I have no idea how github works.  Sorry If I hosed this up.  Also some links to the libraries might need fixing.

### CODE REVIEW

File: examples/css3d_youtube2.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<title>three.js css3d - youtube</title>
+
+	</head>
+	<body>
+		<script src="three.min.js"></script>
```
Comment (mrdoob) [inline examples/css3d_youtube2.html]: This should be `<script src="../build/three.min.js"></script>` instead.

File: examples/css3d_youtube2.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+		<script src="CSS3DRenderer.js"></script>
```
Comment (mrdoob) [inline examples/css3d_youtube2.html]: This should be `<script src="js/renderers/CSS3DRenderer.js"></script>` instead.

**Comment by jsladek** [comment]
I figured out how to make those changes :0)

**Comment by jsladek** [comment]
Not sure if I was supposed to close the pull request.  Sorry, I don't have any idea how this github works.  I'll start learning.

**Comment by mrdoob** [comment]
Almost there! Now, do you mind re-doing the PR but target the `dev` branch instead?

**Comment by jsladek** [comment]
I hope I did this correctly.

**Pull request closed without merging**